lib/config/configItems.joi.js

```diff
@@ -11,6 +11,7 @@ const bootstrapListJoi = joi.array().items(
     joi.object({
         site: joi.string().required(),
         servers: joi.array().items(joi.string()),
+        type: joi.string().allow('aws_s3'),
```

Comment:
This does not prevent other values to be allowed (i.e. any string), you want to use `valid()` instead.

And IMO you should also check in the config parsing function that either `servers` or `type` is provided but not both. (or there may be more or less arcane ways to do this with `joi` directly).

Reply:
An additional style comment: the current `bootstrapListJoi` is CRR-specific, maybe we set the definition of the `servers` array as the `bootstrapListJoi` instead to allow re-use.

Reply:
You caught me being lazy 😄